### PR TITLE
migrate

### PR DESCRIPTION
**Pull request by mihalskiy**

### CODE REVIEW

File: database/migrations/2018_02_26_164116_create_table_pages.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        Schema::create('pages', function (Blueprint $table) {
+            $table->increments('id');
+
+            $table->string('name',100);
+            $table->string('alias',100);
+
+
+            $table->text('description');
+
+            $table->string('images');
```
Comment (odykyi) [inline database/migrations/2018_02_26_164116_create_table_pages.php]: WTF?? images ??
maybe 'imageURL' type string?

File: database/migrations/2018_02_26_164322_create_table_services.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    /*
+     * php artisan make:migration create_table_services --create=services
+    */
+    public function up()
+    {
+        Schema::create('services', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('name');
+            $table->text('description');
+            $table->string('icon',100);
```
Comment (odykyi) [inline database/migrations/2018_02_26_164322_create_table_services.php]: change to `iconURL`

words like a URL, HTTP, REST, CRUD should be in CAPS LOCK CASE

File: database/migrations/2018_02_26_164417_create_table_portfolios.php [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+    /*
+     * php artisan make:migration create_table_portfolios --create=portfolios
+    */
+    public function up()
+    {
+        Schema::create('portfolios', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('name');
+            $table->string('images');
```
Comment (odykyi) [inline database/migrations/2018_02_26_164417_create_table_portfolios.php]: WTF?? images ??
maybe 'imageURL' type string?

File: database/migrations/2018_02_26_164417_create_table_portfolios.php [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+            $table->string('filter');
```
Comment (odykyi) [inline database/migrations/2018_02_26_164417_create_table_portfolios.php]: WTF?? filter ??
change to more detailed column name

File: database/migrations/2018_02_26_164518_create_table_peoples.php [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    /*
+     * php artisan make:migration create_table_peoples --create=peoples
+    */
+    public function up()
+    {
+        Schema::create('peoples', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('name');
+            $table->string('position');
+            $table->string('images');
```
Comment (odykyi) [inline database/migrations/2018_02_26_164518_create_table_peoples.php]: WTF?? images ??
maybe 'imageURL' type string?

File: package.json [hunk 6]
```diff
@@ -8,5 +8,23 @@
     "gulp": "^3.9.1",
     "laravel-elixir": "^5.0.0",
     "bootstrap-sass": "^3.0.0"
-  }
+  },
```
Comment (odykyi) [inline package.json]: write in console 
`npm init -y `

this command will fix `package.json` file

File: database/migrations/2018_02_26_164116_create_table_pages.php [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+    /*
+     * php artisan make:migration create_table_pages --create=pages
+    */
+    public function up()
+    {
+        Schema::create('pages', function (Blueprint $table) {
+            $table->increments('id');
+
+            $table->string('name',100);
+            $table->string('alias',100);
```
Comment (odykyi) [inline database/migrations/2018_02_26_164116_create_table_pages.php]: WTF?? alias ??
not understand

File: package.json (hunk 6 above)
Comment (mihalskiy) [reply package.json]: ok, thanks, i'm fixed this problem

File: database/migrations/2018_02_26_164116_create_table_pages.php [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+    /*
+     * php artisan make:migration create_table_pages --create=pages
+    */
+    public function up()
+    {
+        Schema::create('pages', function (Blueprint $table) {
+            $table->increments('id');
+
+            $table->string('name',100);
+            $table->string('detail',100)->comment('read more');
```
Comment (odykyi) [inline database/migrations/2018_02_26_164116_create_table_pages.php]: detail change to -> short_description 
description change to -> long_description

File: package.json (hunk 6 above)
Comment (odykyi) [reply package.json]: awesome

File: database/migrations/2018_02_26_164116_create_table_pages.php [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+    public function up()
+    {
+        Schema::create('pages', function (Blueprint $table) {
+            $table->increments('id');
+
+            $table->string('name',100);
+            $table->string('long_description',100)->comment('read more');
+
+
+            $table->text('description');
```
Comment (odykyi) [inline database/migrations/2018_02_26_164116_create_table_pages.php]: change to short_description

**Pull request merged**